config/config.go

```diff
@@ -1520,3 +1532,12 @@ func providedFlags(v *viper.Viper) map[string]interface{} {
 	}
 	return customSettings
 }
+
+// getRenamedKey returns the new key if it is set, otherwise it returns the deprecated key.
+func getRenamedKey(v *viper.Viper, deprecatedKey string, newKey string) string {
```

Comment:
Should we error if both are provided? Probably not _strictly_ necessary but we've done this in the past

Reply:
yea I thought about it but then we need to handle error everytime we use it. I think it's fine to just use the new key when both are provided.